_resources/port1.0/group/app-1.0.tcl

```diff
@@ -304,3 +304,18 @@ proc app._resolve_symlink {path destroot} {
 #    ui_debug "In ${destroot}, ${path} is a symlink to ${resolved_path}"
     return [app._resolve_symlink ${resolved_path} ${destroot}]
 }
+
+
+# Write a default launch script for the executable into the bundle, 
+# setting the default PATH as would be expected by the binary
+proc app._write_launch_script  {executable app_destination} {
+    global prefix
+    set launch_script [open ${app_destination} w]
+
+    puts ${launch_script} "#!/bin/bash
+export PATH=\"${prefix}/bin:${prefix}/sbin:\$PATH\"
+eval ${executable}
```

Comment:
I think you meant `exec` instead of `eval`.

Reply:
I did, thanks -- but it turns out eval works as well.